apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/RequestResponsePopulateMdcFilter.java

```diff
@@ -146,30 +146,31 @@ private static X509Certificate getClientCertificate(HttpServletRequest request)
   }
 
   /**
-   * @param request the {@link ServletRequest} to record the standard {@link MDC} entries for
-   * @param response the {@link ServletResponse} to record the standard {@link MDC} entries for
+   * @param request the {@link ServletRequest} to record the standard {@link BfdMDC} entries for
+   * @param response the {@link ServletResponse} to record the standard {@link BfdMDC} entries for
    */
   private void handleResponse(ServletRequest request, ServletResponse response) {
     if (response instanceof HttpServletResponse) {
       HttpServletResponse servletResponse = (HttpServletResponse) response;
 
-      MDC.put(computeMdcKey("response.status"), Integer.toString(servletResponse.getStatus()));
+      BfdMDC.put(computeMdcKey("response.status"), Integer.toString(servletResponse.getStatus()));
```

Comment:
general comment - I think we should go ahead and change all of the dots to underscores but still leave the wrapper functionality as a fail-safe